deploy/cloudwash_job.yaml

```diff
@@ -56,16 +57,30 @@ objects:
                       secretKeyRef:
                         key: subscription_id
                         name: provisioning-azure-auth-secret
-                  - name: CLEANUP_AZURE__AUTH__RESOURCE_GROUPS
+              - name: provisioning-aws-cleanup
+                image: quay.io/redhatqe/cloudwash
+                imagePullPolicy: Always
+                command:
+                  - swach
+                args:
+                  - "-d" # Need confirmation from jakub or akhil then only we can remove this flag.
+                  - "aws"
+                  - "--all"
```

Comment:
Only vms for aws and gcp. 
The structure of azure vs aws/gcp is different on how we are billed.

Reply:
and Yes, we can remove the `-d` parameter.

Reply:
We don't care about the vm prefix for these clouds.

Reply:
added only vms flag and removed -d flag 
thanks!